ParallelEngine/entity.h

```diff
@@ -38,15 +38,12 @@ class Entity {
 	void renderRadius();
 	void moveTo(int x, int y);
 
-	void setcol(float col_) { col = col_; }
-	void setrow(float row_) { row = row_; }
-
 	float getcol() const { 
-		return row;
+		return col;
 	}
```

Comment:
Any reason for not using camelCase for these?

Reply:
With smaller variable names I think using camelCase is awkward. Im keeping consistency since the row and col are related to the x and y, and those getters are not camel case either. (getx() and gety())